rune/libenclave/internal/runtime/pal/skeleton/liberpal-skeleton-v2.c

```diff
@@ -13,17 +13,22 @@ int pal_get_version(void)
 
 int pal_init(pal_attr_t *attr)
 {
-	if (is_oot_driver) {
-		fprintf(stderr, "Skeleton PAL API v2 doesn't support SGX OOT driver!\n");
-		return -1;
-	}
-
 	return __pal_init(attr);
 }
 
 int pal_create_process(pal_create_process_args *args)
 {
-	return __pal_create_process(args);
+	if (args == NULL || args->path == NULL || args->argv == NULL || args->pid == NULL || args->stdio == NULL) {
+		return -1;
+	}
+
+	if (!is_oot_driver) {
+		return __pal_create_process(args);
+	} else {
+		int exit_code;
+
+		return __pal_exec(args->path, args->argv, args->stdio, &exit_code);
+	}
```

Comment:
I don't see you do the modifications to pal_exec() for oot driver case. In that case, pal_exec will call wait4child() and never return?